src/SprykerEco/Yves/Computop/Plugin/Router/ComputopRouteProviderPlugin.php

```diff
@@ -194,6 +196,24 @@ protected function addSofortSuccessRoute(RouteCollection $routeCollection): Rout
         return $routeCollection;
     }
 
+    /**
```

Comment:
* Add the `@uses` document block that points to the gateway controller.
* RFC - https://spryker.atlassian.net/wiki/spaces/RFC/pages/991985877.

Reply:
Done